docs/dashboard/apps.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Apps
+---
+
+This page gives you an overview of how to manage apps in the dashboard.
+
+You can find the _Apps_ section on the main page of the dashboard. The Apps page allows you to install and manage your apps.
+
+The main page of Apps contains the following sections:
+
+- _Third-Party Apps_ a list of installed third-party apps in your dashboard.
+
+- _Local Apps_ with an option to create a new app.
+
+- _Saleor Marketplace_ - coming in Q3/Q4 2020.
+
+![](screenshots/apps.jpg)
```

Comment:
Not sure of the screenshot. The name of local app is not meaningfull and it looks like the email `maciej@mirumee SA`.
Something like this would be better:
![image](https://user-images.githubusercontent.com/23282005/93563562-714d9600-f988-11ea-820f-0548a4a6ae7a.png)

Reply:
Also all new screenshots show the new design of dashboard. It will not be released in 2.11 so I think that we should replace them by screenshots of dashboard 2.11